website/src/components/HomepageFeatures/index.tsx

```diff
@@ -1,22 +1,29 @@
+import { useColorMode } from "@docusaurus/theme-common";
 import Heading from "@theme/Heading";
 import clsx from "clsx";
 
 import boxImage from "../../../static/img/box.png";
+import boxDarkImage from "../../../static/img/box_dark.png";
 import chainImage from "../../../static/img/chain.png";
+import chainDarkImage from "../../../static/img/chain_dark.png";
 import chartImage from "../../../static/img/chart.png";
+import chartDarkImage from "../../../static/img/chart_dark.png";
 
 import styles from "./styles.module.css";
 
 interface FeatureItem {
   title: string;
   img: string;
+  // eslint-disable-next-line react/no-unused-prop-types
```

Comment:
Please, don't disable linter rules. I think this one can be easily removed because you use this prop.

Reply:
I think the link check failed of this, not sure how to rewrite this part could make it clearer

Reply:
So, you need to use props type for `Feature` component `Omit<FeatureItem, "imgDark">`. This is because this component doesn't use `imgDark` but it's defined in props.

Reply:
yep right, but I move the decision on whether to load dark or light image to that component, so I think it looks fine now.